programs/makepkg/meta.yaml

```diff
@@ -0,0 +1,12 @@
+short_description: >
+ fakeroot can be built to either use SYSV IPC or TCP for message passing.
```

Comment:
thanks, no need to add short description, ok to leave empty

Reply:
ok